internal/healthchecks/error.go

```diff
@@ -141,6 +145,22 @@ var (
 		ResourceLink: "https://cloud.google.com/monitoring/api/enable-api",
 		IsFatal:      true,
 	}
+	LogApiUnauthenticatedErr = HealthCheckError{
+		Code:         "LogApiUnauthenticatedErr",
+		Class:        Api,
+		Message:      "The current VM couldn't authenticate to the Logging API.",
+		Action:       "Verify that your credential files, scopes and permissions are setup correctly.",
```

Comment:
"are set up correctly"  ("setup" as one word is a noun)

Reply:
Fixed.